politeiavoter/politeiavoter.go

```diff
@@ -188,7 +189,9 @@ func newClient(cfg *config) (*ctx, error) {
 		client: &http.Client{
 			Transport: tr,
 			Jar:       jar,
-		}}, nil
+		},
+		userAgent: fmt.Sprintf("politeiavoter %s", cfg.Version),
```

Comment:
User agent formatting should be `politeiavoter/<version>` per [RFC7231](https://tools.ietf.org/html/rfc7231#section-5.5.3).

Reply:
thanks